Packs/ExabeamDataLake/Integrations/ExabeamDataLake/ExabeamDataLake.py

```diff
@@ -0,0 +1,273 @@
+
+    dates = []
+    current_date = start_time
+    while current_date <= end_time:
+        dates.append(current_date.strftime("%Y.%m.%d"))
+        current_date += timedelta(days=1)
+
+    return dates
+
+
+def get_date(time: str):
+    """
+    Get the date from a given time string.
+
+    Args:
+        time (str): The time string to extract the date from.
+
+    Returns:
+        str: The date extracted from the time string formatted in ISO 8601 format (YYYY-MM-DD),
+        or None if the time string is invalid.
+    """
+    date_time = arg_to_datetime(arg=time, arg_name="Start time", required=True)
+    if date_time:
+        date = date_time.strftime(ISO_8601_FORMAT)
+    return date
+
+
+def calculate_page_parameters(args: dict) -> tuple[int, int]:
+    """
+      Calculate the page parameters for pagination.
+
+    Args:
+        args: A dictionary containing the arguments passed to the function.
+
+    Raises:
+        DemistoException: If invalid combinations of arguments are provided. You can only provide 'limit'
+        alone or 'page' and 'page_size' together.
+
+    Returns:
+        tuple: A tuple containing two integers representing the 'from' and 'size' parameters for pagination.
+        'from' is the index of the first item to retrieve, and 'size' is the number of items to retrieve.
+    """
+    page_arg = args.get('page')
+    page_size_arg = args.get('page_size')
+    limit_arg = args.get('limit')
+
+    if (limit_arg and (page_arg or page_size_arg)) or ((not (page_arg and page_size_arg)) and (page_arg or page_size_arg)):
+        raise DemistoException("You can only provide 'limit' alone or 'page' and 'page_size' together.")
+
+    if args.get('page') and args.get('page_size'):
+        page = arg_to_number(args.get('page', '1'))
+        page_size = arg_to_number(args.get('page_size', '50'))
+        if page and page_size:
+            from_param = page * page_size - page_size
+            size_param = page_size
+    else:
+        from_param = 0
+        size_param = arg_to_number(args.get('limit', '50')) or 50
+
+    return from_param, size_param
+
+
+""" COMMAND FUNCTIONS """
+
+
+def query_datalake_command(client: Client, args: dict, cluster_name: str) -> CommandResults:
+    """
+    Query the datalake command and return the results in a formatted table.
+
+    Args:
+        client: The client object for interacting with the API.
+        args: The arguments passed to the command.
+
+    Returns:
+        CommandResults: The command results object containing outputs and readable output.
+    """
+    from_param, size_param = calculate_page_parameters(args)
+
+    start_time = get_date(args.get("start_time", "7 days ago"))
+    end_time = get_date(args.get("end_time", "today"))
+    dates = dates_in_range(start_time, end_time)
+    dates_in_format = ["exabeam-" + date for date in dates]
+
+    search_query = {
```

Comment:
please move this insice the client query_datalake_request it makes more sense to go there. 
Also if you call the command 
exabeam-data-lake-search
change the function name to 
query_data_lake_command